flink/v1.13/flink/src/main/java/org/apache/iceberg/flink/sink/IcebergStreamWriter.java

```diff
@@ -100,4 +100,4 @@ public String toString() {
   private void emit(WriteResult result) {
     output.collect(new StreamRecord<>(result));
   }
-}
+}
```

Comment:
please revert unnecessary blank line change for 1.13 and 1.14

Reply:
Thank you, I have made the modification.